src/widget/widget.cpp

```diff
@@ -144,7 +144,10 @@ void Widget::init()
     actionLogout->setIcon(prepareIcon(":/img/others/logout-icon.svg", icon_size, icon_size));
 
     actionQuit = new QAction(this);
+#ifndef Q_OS_OSX
     actionQuit->setMenuRole(QAction::QuitRole);
+#endif
+    actionQuit->setShortcut(QKeySequence::Quit);
```

Comment:
afair @antis81 had better idea of global quit button? @antis81 you ok with merging it?